torch/_inductor/sizevars.py

```diff
@@ -356,6 +356,8 @@ def statically_known_multiple_of(
         """
         Return a bool indicating if it is sound to optimize for the numerator being a multiple of the denominator.
         """
+        if free_unbacked_symbols(numerator) or free_unbacked_symbols(denominator):
```

Comment:
i'm a little worried about how `torch._dynamo.decorators.mark_unbacked(x, 0)` will play with this, but I think the answer is that with that API we still provide a hint, and so we wouldn't actually end up with any free unbacked symbols

Reply:
It's OK. We will simply assume that it is not aligned!